app/frontend/js/map.js

```diff
@@ -59,21 +50,41 @@ const createProjection = () => {
 export function displayMap (sbi, parcels, coordinates) {
   const features = new GeoJSON().readFeatures(parcels)
   const parcelSource = new VectorSource({ features })
-  const parcelLayer = new VectorLayer({ source: parcelSource, style: styleFunction })
-  const baseLayer = new TileLayer({ preload: Infinity, source: createBingMapsSource() })
+  const parcelLayer = new VectorLayer({ source: parcelSource, style: styleFunction, visible: true })
   const projection = createProjection()
 
+  const layers = []
+
+  const mapStyles = [
+    'RoadOnDemand',
+    'Aerial',
+    'AerialWithLabelsOnDemand',
+    'CanvasDark',
+    'OrdnanceSurvey']
+
+  for (let i = 0, ii = mapStyles.length; i < ii; ++i) {
```

Comment:
The `ii` variable should be declared outside the loop definition, like the other loop